internal/server/adapter/pgsqlrepo/userrepo/get_user_by_email.go

```diff
@@ -0,0 +1,21 @@
+package userrepo
```

Comment:
```suggestion
package user
```
тут вся директория относится к репозиторию, нет смысла добавлять префиксы/постфиксы

Reply:
++